Copyright (c) 2009, 2010, 2013-2016 by the Brotli Authors.

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in
all copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
THE SOFTWARE.

Date: Aug 23 14:44:26 2016

From: eace82070f519ed86a2869dbdfe437f407ef282d Aug 24 11:04:58 2016

Subject: [PATCH] RDK-16104 : Develop code - Brotli

        Enable brotli and solve compile time issues

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: Vargheese Baby <vargheesebaby@tataelxsi.com>
---

Index: git/setup.py
===================================================================
--- git.orig/setup.py
+++ git/setup.py
@@ -120,7 +120,7 @@ class BuildExt(build_ext):
 
 brotli = Extension("brotli",
                     sources=[
-                        "python/brotlimodule.cc",
+                        "enc/compressor.cc",
                         "common/dictionary.c",
                         "dec/bit_reader.c",
                         "dec/decode.c",
